src/Scripting/VM/VirtualMachine.cpp

```
@@ -0,0 +1,1281 @@
+					Compiler::StackValueToByte(typeid(ValueType))
+				)
+			);
+		}
+	}, val);
+
+	return stackVal;
+}
+
+
+FORCE_INLINE void VirtualMachine::saveVMState(std::optional<Compiler::VMExitCode> exitCode) {
+	if (exitCode.has_value())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)